content/en/04-tutorials/01-pixie-101/06-profiler.md

```diff
@@ -9,14 +9,30 @@ redirect_from:
 
 This tutorial will demonstrate how to use Pixie's Always-On Profiling feature to investigate a spike in CPU utilization, using a flamegraph to identify a performance issue within the application code.
 
-Pixie's profiling feature currently supports Go, C++, and Rust. Other language support coming soon.
+Pixie's continuous profiler currently supports Go, C++, and Rust. Java support is a beta feature and must be [manually enabled](#prerequisites-enabling-java-support). Other language support coming soon.
```

Comment:
It is not beta, but just not enabled by default, so let's change the wording to "Java support must be manually enabled..."